src/sgame/sg_spawn_shared.cpp

```diff
@@ -69,6 +69,14 @@ void think_aimAtTarget( gentity_t *self )
 	VectorAdd( self->r.absmin, self->r.absmax, origin );
 	VectorScale( origin, 0.5f, origin );
 
+	if ( false
```

Comment:
why

Reply:
old trick of mine. Forgot to remove it from unv.
I usually do this when I have a `if` with long lines. This makes it possible to:

1. `yyp` in vim to quickly duplicate stuff,
2. alignment is made with tabs or space: don't care, it just works,
3. you can remove or add components of the if by touching a single line, reduce number of changes in git

Reply:
Note that this would not have happened with a sane interface, instead of that horribly vec3_t.